### ISSUE TITLE
classification loss

### ISSUE THREAD

**Issue by ConanMusk** [eduardzamfir/DaAIR#3]
![image](https://github.com/eduardzamfir/DaAIR/assets/67884112/2b2e3140-a361-46c5-a424-fbc1a3ffe02d)
I am very curious about the classification loss function you mentioned here, and I wonder if this requires adding labels to the training data? Which article did you get this idea from? I would appreciate it if you could reply to my questions.

**Comment by eduardzamfir** [comment]
Hey,

Thanks for your interest. 

We utilize the labels inherent in our datasets and encode them as one-hot vectors, as these labels are available during training. During inference, this additional information is not required. Although there is no direct link to previous research, our aim is for each degradation expert to specialize in a specific task.
